src/components/ExitButton.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Close({ isOpen, exitModal }) {
+  return (
+    <div>
+      <button type="button" className="exit" onClick={() => { this.exitModal}}>
```

Comment:
does that work when you use the onClick that way?  I would have just used {() => { exitModal() }}.

Reply:
This part is not functional yet, so probably not